docs/scripts/api/buildSelectorsDocumentation.ts

```diff
@@ -19,6 +19,7 @@ interface Selector {
   category?: string;
   deprecated?: string;
   description?: string;
+  supportsApiRef?: boolean;
```

Comment:
Why some selectors do not support `apiRef` signature?

Reply:
Because they are only a simple functions that return part of the state. They don't cache the result. Currently, we don't document them but this file is ready if we decide to provide docs for them.

https://github.com/mui-org/material-ui-x/blob/f6469fbeb3eac53b75792d7eb868d6358b2676a1/packages/grid/_modules_/grid/hooks/features/editRows/gridEditRowsSelector.ts#L4

Reply:
Right, I didn't notice that those are not using `createSelector`